src/scripts/towns/TownList.ts

```diff
@@ -2597,13 +2633,58 @@ const SunyshoreRibbonerJulia = new NPC('Ribboner Julia', [
+});
+
+const FightAreaZero2 = new NPC('Zero', [
+    'What was in that book? Did you find any information on the Distortion World?',
+    'It didn\'t mention it once? That\'s a shame.',
+    'Wait, you said the book mentioned the Lake Trio, right? Legends say they have existed since the creation of the world, they may know how to enter the Distortion World.',
+], {
+    image: 'assets/images/temporaryBattle/Zero.png',
+    requirement: new MultiRequirement([new QuestLineStepCompletedRequirement('Zero\'s Ambition', 1), new QuestLineStepCompletedRequirement('Zero\'s Ambition', 2, GameConstants.AchievementOption.less)]),
+});
+
 const SurvivalAreaSinnohRoamerNPC = new RoamerNPC('Hiker Kevin', [
     'I spotted a bunch of roaming Pokémon on {ROUTE_NAME}!',
 ], GameConstants.Region.sinnoh, RoamingPokemonList.findGroup(GameConstants.Region.sinnoh, GameConstants.SinnohSubRegions.Sinnoh), 'assets/images/trainers/Hiker.png');
 
+const SendoffSpringLakeTrio = new NPC('Lake Trio', [
+    '<i>You are finally here.</i>',
+    '<i>We have been able to create a key to the Distortion World using the materials you collected, but it is unable to work in it\'s current state.</i>',
+    '<i>We bestow it upon you, charge it with the ghostly energy of the Distortion World.</i>',
+    '<i>In order to do so, you must collect Ghost gems by defeating Ghost type Pokémon and find a Spooky Plate. There should be some in this dungeon, we can sense their energy.</i>',
+],{
+    image: 'assets/images/npcs/Lake Trio.png',
+    requirement: new MultiRequirement([new QuestLineStepCompletedRequirement('Zero\'s Ambition', 9), new QuestLineStepCompletedRequirement('Zero\'s Ambition', 11, GameConstants.AchievementOption.less)])});
+
+const SendoffSpringZero1 = new NPC('Zero', [
+    'I couldn\'t wait anymore for you to report about your progress, so I asked around and discovered you were coming here.',
+    'Wait, is that the key to the Distortion World? With that, I could open the gate myself.',
+    'You know what? You are fired, I no longer need you. Now, give me the key to the Distortion World or I will take it by force!',
+], {
+    image: 'assets/images/temporaryBattle/Zero.png',
+    requirement: new MultiRequirement([new QuestLineStepCompletedRequirement('Zero\'s Ambition', 11), new QuestLineStepCompletedRequirement('Zero\'s Ambition', 12, GameConstants.AchievementOption.less)]),
+});
+
+const SendoffSpringZero2 = new NPC('Zero', [
+    'Seems like you have managed to catch the Pokémon that rules the Distortion World.',
+    'But this doesn\'t look like the Renegade Pokémon from legend, it seems like some sort of altered form.',
+    'I once read that, deep within the Distortion World, there is an object capable of carrying the realm\'s power. Maybe, if you train Giratina while it has such object equipped, it can turn into it\'s true form.',
```

Comment:
```suggestion
    'I once read that, deep within the Distortion World, there is an object capable of carrying the realm\'s power. Maybe, if you train Giratina while it has such object equipped, it can turn into its true form.',
```